mrmustard/lab_dev/circuit_components_utils/b_to_q.py

```diff
@@ -29,13 +29,27 @@
 
 class BtoQ(Operation):
     r"""
-    The Operation that changes the representation of an object from ``Bargmann`` into quadrature.
-    By default it's defined on the output ket side. Note that beyond such gate we cannot place further
-    ones unless they support inner products in quadrature representation.
+    The Operation that changes the representation of an object from Bargmann (B) into quadrature (Q).
```

Comment:
```suggestion
    The operation that changes the representation of an object from Bargmann (B) into quadrature (Q).
```
Unless operation here is refering to the object in which case
```suggestion
    The ``Operation`` that changes the representation of an object from Bargmann (B) into quadrature (Q).
```
is more appropriate